docs/Server-Methods.md

```diff
@@ -25,6 +25,17 @@ fastify.ready().then(() => {
 })
 ```
 
+<a name="after"></a>
+### after
+An function called when all routes have been loaded
```

Comment:
"Invoked when all routes..."

Reply:
"Invoked when all plugins and routes ..."